README.md

```diff
@@ -263,6 +263,7 @@ Contributions are welcome. Please take a quick look at the [contribution guideli
  * [graphql-crystal](https://github.com/ziprandom/graphql-crystal) - [Graphql](http://graphql.org) implementation
  * [kave](https://github.com/jwoertink/kave) - Kemal API Version Extension
  * [kemal-session](https://github.com/kemalcr/kemal-session) - Session handler for Kemal
+ * [mochi](https://github.com/gitlato/mochi) - Authentication shard with authentication, confirmation, oauth, invites & more (inspired by devise)
```

Comment:
"Authentication shard with authentication" sounds strange. Would you mind to re-phase it.

Reply:
Lol yeah that sounded pretty bad.